automation/features/step-definitions/staking.steps.js

```diff
@@ -48,23 +34,228 @@ When(/^I click on use maximum button$/, () => {
     reverse: false,
     timeoutMsg: "callout was not found",
   });
-  if (browser.getByTestId("associate-more-tokens-btn").isDisplayed()) {
-    browser.getByTestId("associate-more-tokens-btn").click();
-  } else {
-    browser.getByTestId("associate-tokens-btn").click();
-  }
+  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+    stakingPage.associateMoreTokensBtn.click();
+  } else stakingPage.associateTokensBtn.click();
   $('[data-testid="associate-radio-wallet"]').click({ force: true });
-  currentAmountOfTokensInWallet = $(".wallet-card__asset-balance").getText();
+  currentAmountOfTokensInWallet = browser
+    .getByTestId("eth-wallet-balance")
+    .getText();
   browser.getByTestId("token-amount-use-maximum").click();
 });
 
 When(
   /^I can see the maximum amount of tokens in my wallet are in the token input box$/,
   () => {
     const noZeroes = currentAmountOfTokensInWallet.toString();
-    const noZeroesFloat = parseFloat(noZeroes);
+    const noZeroesFloat = parseFloat(noZeroes.replace(",", ""));
```

Comment:
Variable names not what they suggest. These have nothing to do with zeros. Suggest `currentBalance`